plenum/config.py

```diff
@@ -322,3 +322,7 @@
 METRICS_KV_STORAGE = KeyValueStorageType.Rocksdb
 METRICS_KV_DB_NAME = 'metrics_db'
 METRICS_KV_CONFIG = rocksdb_default_config.copy()
+
+ACC_MONITOR_ENABLED = False
+ACC_MONITOR_THRESHOLD = 100  # transactions delta
+ACC_MONITOR_TIMEOUT = 900  # seconds
```

Comment:
Maybe `ACC_MONITOR_DEGRADATION_WINDOW` is a better name? It would also be nice to add a descriptive comment what it means.